metals/src/main/scala/scala/meta/internal/metals/MetalsLspService.scala

```diff
@@ -168,8 +173,13 @@ class MetalsLspService(
     initializeParams,
   )
 
-  private val languageClient =
-    new ConfiguredLanguageClient(client, clientConfig)
+  private val languageClient = {
+    val languageClient = new ConfiguredLanguageClient(client, clientConfig)
+    // Set the language client so that we can forward log messages to the client
+    LanguageClientLogger.languageClient = Some(languageClient)
+    cancelables.add(() => languageClient.shutdown())
+    languageClient
+  }
```

Comment:
Ok, I think this is how it was done previously:
1. Use `ConfiguredLanguageClient` in metals lsp service
2. shutdown `ConfiguredLanguageClient` instance (start being noop)
3. Do not shut down proxy from lsp4j